CMakeLists.txt

```diff
@@ -256,7 +260,10 @@ if(WIN32)
   target_compile_definitions(srtp2 PUBLIC _CRT_SECURE_NO_WARNINGS)
 endif()
 
-install(TARGETS srtp2 DESTINATION lib)
+install(TARGETS srtp2 DESTINATION lib
```

Comment:
Here should be:
```suggestion
install(TARGETS srtp2 EXPORT libSRTPTargets
        RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
)
```

Reply:
This is a different change, current libsrtp installs to lib regardless if it is shared or static.